### PR TITLE
Update gradle to latest with new dependency configs. #8

### PR DESCRIPTION
**Pull request by lemonboston**

### CODE REVIEW

File: color-bolts/build.gradle [hunk 1]
```diff
@@ -2,7 +2,7 @@ apply plugin: 'com.android.library'
 
 android {
     compileSdkVersion 25
```
Comment (dmfs) [inline color-bolts/build.gradle:4]: The `compileSdkVersion` should be in sync with the `BUILD_TOOLS_VERSION`, so better create a property for this one as well.

Comment (lemonboston) [reply color-bolts/build.gradle:4]: I've just checked documentation:
https://developer.android.com/studio/build/index.html

```groovy
  /**
   * compileSdkVersion specifies the Android API level Gradle should use to
   * compile your app. This means your app can use the API features included in
   * this API level and lower.
   */

  compileSdkVersion 26

  /**
   * buildToolsVersion specifies the version of the SDK build tools, command-line
   * utilities, and compiler that Gradle should use to build your app. You need to
   * download the build tools using the SDK Manager.
   *
   * If you're using Android plugin 3.0.0 or higher, this property is optional—
   * the plugin uses a recommended version of the build tools by default.
   */

  buildToolsVersion "26.0.2"
```
So it seems for me that `compileSdkVersion` should be in sync with target level, shouldn't  it?
But the comment at `buildToolsVersion` that it became optional means that we should probably just remove it. I am going to push a commit.

Comment (lemonboston) [reply color-bolts/build.gradle:4]: Pushed, ready for check.

Comment (dmfs) [reply color-bolts/build.gradle:4]: Right, I noticed that myself later on.

**Review by dmfs — APPROVED** [review]

**Pull request merged**